onnxscript/function_libs/torch_lib/graph_building.py

```diff
@@ -342,6 +343,18 @@ def _create_op_call_in_torch_graph(
     return node_ouputs
 
 
+def _estimate_tensor_size(tensor: torch.Tensor) -> int:
```

Comment:
```suggestion
def _tensor_rawdata_size(tensor: torch.Tensor) -> int:
```
would this be more accurate?

Reply:
Done